include/z64actor.h

```diff
@@ -574,6 +574,76 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+typedef struct TargetContextEntry {
+    /* 0x00 */ Vec3f    pos;
+    /* 0x0C */ f32      unk_0C; // radius?
+    /* 0x10 */ Color_RGB8 color;
+} TargetContextEntry; // size = 0x14
+
+typedef struct TargetContext {
+    /* 0x00 */ Vec3f    naviRefPos; // possibly wrong
+    /* 0x0C */ Vec3f    targetCenterPos;
```

Comment:
Since you are moving these structs anyways, might as well update the formatting since we no longer have the members aligned really.